packages/server/graphql/mutations/helpers/handleCompletedStage.ts

```diff
@@ -73,6 +74,7 @@ const handleCompletedRetrospectiveStage = async (
         })
         .run()
       data.meeting = meeting
+      generateGroupSummaries(meeting.id, dataLoader, facilitatorUserId)
```

Comment:
We're not awaiting here so that the meeting doesn't wait for the OpenAI API when proceeding to the Vote stage. It's possible that the user could race through the Vote & Discuss stage, reach the meeting summary, and not see the GPT-3 topic summary as the API hasn't responded in time. 

I think that's fine. They can always refresh the page once the API has responded and they'll then see the topic summary.